spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
```

Comment:
- [ ] As required for the project, kindly ensure that you use `type: request` instead of `type: :controller` in each controller spec file OR use `config.infer_spec_type_from_file_location!` in Rspec config file